patches/makepkg.conf

```diff
@@ -0,0 +1,151 @@
+#!/hint/bash
+#
+# /usr/local/pspdev/etc/makepkg.conf
+#
+#########################################################################
+# SOURCE ACQUISITION
+#########################################################################
+#
+#-- The download utilities that makepkg should use to acquire sources
+#  Format: 'protocol::agent'
+DLAGENTS=('file::/usr/bin/curl -gqC - -o %o %u'
+          'ftp::/usr/bin/curl -gqfC - --ftp-pasv --retry 3 --retry-delay 3 -o %o %u'
+          'http::/usr/bin/curl -gqb "" -fLC - --retry 3 --retry-delay 3 -o %o %u'
+          'https::/usr/bin/curl -gqb "" -fLC - --retry 3 --retry-delay 3 -o %o %u'
+          'rsync::/usr/bin/rsync --no-motd -z %u %o'
+          'scp::/usr/bin/scp -C %u %o')
+
+# Other common tools:
+# /usr/bin/snarf
+# /usr/bin/lftpget -c
+# /usr/bin/wget
+
+#-- The package required by makepkg to download VCS sources
+#  Format: 'protocol::package'
+VCSCLIENTS=('bzr::bzr'
+            'git::git'
+            'hg::mercurial'
+            'svn::subversion')
+
+#########################################################################
+# ARCHITECTURE, COMPILE FLAGS
+#########################################################################
+#
+AR="psp-ar"
+RANLIB="psp-ranlib"
+CC="psp-gcc"
+CXX="psp-g++"
+CARCH="mips"
+CHOST="psp"
+
+# Cmake args for when you're too lazy to use a toolchain file : TODO - don't be lazy!
+CMARGS="-DCMAKE_CXX_LINK_EXECUTABLE=$(which $CXX) -DCMAKE_CC_LINK_EXECUTABLE=$(which $CC) -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_SYSTEM_NAME=Generic -DCMAKE_C_COMPILER=$CC -DCMAKE_CXX_COMPILER=$CXX -DCMAKE_RANLIB=$(which $RANLIB) -DCMAKE_AR=$(which $AR) -DCMAKE_FIND_ROOT_PATH=$PREFIX -DCMAKE_FIND_ROOT_PATH_MODE_LIBRARY=ONLY -DCMAKE_FIND_ROOT_PATH_MODE_INCLUDE=ONLY -DCMAKE_FIND_ROOT_PATH_MODE_PROGRAM=NEVER"
```

Comment:
We do have a toolchain file → remove this.

Reply:
Done! I got this from the vita makepkg config sample, but I guess we'd never need it.